airbyte-connector-builder-server/build.gradle

```diff
@@ -42,4 +42,18 @@ task generateOpenApiPythonServer(type: GenerateTask){
 }
 
 project.build.dependsOn(generateOpenApiPythonServer)
-    
+
+task prepareBuild(type: Copy) {
```

Comment:
why is this needed? can you add a comment with the context?

Reply:
I left a comment [here](https://github.com/airbytehq/airbyte/pull/19309#pullrequestreview-1176626476) asking if this step meant that if in the future we add some new file or directory that we need to be in the docker image, then we will need to update this step as well. And you said that you could replace this with just a `"."` to copy everything in the directory over.

Just curious, is there a reason you didn't go with that approach here? (Also, is that other PR being replaced with this one?)

Reply:
copy "." copies a bunch unwanted hidden files. Will look for an exclude filter because i still agree with you

Reply:
added an exclude filter